### DISCUSSION TITLE
Testing Livewire V3 results in DuskBrowserMacros class not found

### DISCUSSION THREAD

**Discussion by schonhoff** [livewire/livewire#5808] [7. Bugs (archived)] [answered]
Hi,

first of all thanks for the great update! Really good stuff :-)

I updated my app and tried out my tests. After I tried out a Livewire test it failed.

Here the test:
```php
    public function cannot_render_as_guest(): void
    {
        Livewire::test(AuthorizedClients::class)
            ->assertForbidden();
    }
```

Here the error:
```cmd
1) Tests\Feature\Livewire\AuthorizedClientsTest::cannot_render_as_guest
Error: Class "Tests\DuskBrowserMacros" not found
\vendor\livewire\livewire\src\Features\SupportTesting\DuskTestable.php:51
\vendor\livewire\livewire\src\Features\SupportTesting\SupportTesting.php:17
[...]
```

I don't know why the class couldn't be found. I tried `composer dump-autoload` and removing the vendor directory completely, but it doesn't work. I also tried to switch the namespace from the DuskBrowserMacros in `DuskTestable.php:51` to `\LegacyTests\Browser\DuskBrowserMacros` this also fails. Maybe someone can help?

### Specs
PHP Version: 8.1.9
Laravel Version: 10.15.0
Pest PHP Version: 2.9.1 (I'm using Pest to run the test but it is a lagacy feature test)
Operating System: Windows 11
Livewire Version: 3.0.0-beta.2

**Comment by joshhanley** [comment]
@schonhoff thanks for reporting! We were still publishing our test files to packagist, but that has been fixed in PR #5788 so should be fixed in the next release beta.3. Can you test it then and let us know how it goes? Thanks!

**Reply by nuernbergerA** [reply, marked as answer]
this should fix it
#5872
